docs/tools-reference/custom-kernels-distros/install-alpine-linux-on-your-linode.md

```diff
@@ -2,237 +2,305 @@
         mkdir boot
         mount /dev/sda /alpine/boot
 
-### Download initial APK tools
+### Download APK Tools
 
-1.  Pick your desired release.  In most cases, you can use the latest-stable release, located at (https://nl.alpinelinux.org/alpine/latest-stable/)[https://nl.alpinelinux.org/alpine/latest-stable/]
+1.  Select your desired Alpine Linux release. In most cases, you can use the [latest stable release](http://nl.alpinelinux.org/alpine/latest-stable/)
 
-2.  Identify the current version of the `apk-tools-static` package.  You will need to navigate into the `main/x86_64` directory of your chosen release.  For example, the latest stable would be (https://nl.alpinelinux.org/alpine/latest-stable/main/x86_64/)[https://nl.alpinelinux.org/alpine/latest-stable/main/x86_64/].  From there, simply search for `apk-tools-static`.  Once you have found it, you will need to copy the file's location.  (Right click, Copy Link Address)
+2.  Update the CA Certificates. Finnix (the operating system running in rescue mode) doesn't have them by default, and so `curl` will fail to download the `apk-tools-static` package if you are using HTTPS, as it won't be able to verify the SSL certificate.
 
-3.  Update the CA Certificates.  Finnix doesn't have them by default, and so `curl` will fail to download the `apk-tools-static` package if you are using https, as it won't be able to verify the ssl certificate.
         update-ca-certificates
 
-4.  Download and extract the `apk-tools-static` package.  You should still be in `/alpine`
-        curl -s `apk-tools-static.apk` | tar xz
+3.  Identify the current version of the `apk-tools-static` package. You will need to navigate into the `main/x86_64` directory of your chosen release in your web browser. 
+
+    For example, the latest stable version's `apk-tools-static` package can be found by visiting `http://nl.alpinelinux.org/alpine/latest-stable/main/x86_64/`. From there, simply search for `apk-tools-static`. Once you have found it, you will need to copy the file's location. To do this in most browsers, right click the filename and select **Copy Link Address**.
+
+4.  Download and extract the `apk-tools-static` package to your Linode. You should still be working in the `/alpine` directory when performing this step. Replace `address` in the following command with the address you copied in the previous step:
+        
+        curl -s address | tar xz
+
+5.  Perform the initial distro installation. This will use the latest stable build of Alpine:
 
-5.  Perform initial distro install.  This will preform the install using the latest stable build of Alpine.
-        ./sbin/apk.static --repository https://nl.alpinelinux.org/alpine/latest-stable/main/ --update-cache --allow-untrusted --root /alpine --initdb add alpine-base alpine-mirrors
+        ./sbin/apk.static --repository http://nl.alpinelinux.org/alpine/latest-stable/main/ --update-cache --allow-untrusted --root /alpine --initdb add alpine-base alpine-mirrors
```

Comment:
Looking at this again, one of the things is that the repository url is supposed to be the url they picked earlier, should this get changed to reflect that, like the download apk-tools-static stuff?  Also, it got reverted to http at some point, would be better to be https.